app/src/main/java/de/westnordost/streetcomplete/quests/toilets_fee/AddToiletsFee.java

```diff
@@ -0,0 +1,51 @@
+package de.westnordost.streetcomplete.quests.toilets_fee;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddToiletsFee extends SimpleOverpassQuestType
+{
+	@Inject public AddToiletsFee(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways, relations with amenity=toilets and !fee";
```

Comment:
amenity=toilets is documented that it should not be used on relations. That https://www.openstreetmap.org/relation/3005219 is a relation makes no sense, was the user trying to "save" geometry by reusing lines?!
Anyhow, no reason to make the overpass query more complex because 0.05% of all toilets are relations. After all, by only processing reasonable tags, the app helps to promote data quality as well.